.gitignore

```diff
@@ -26,5 +26,6 @@ coverage
 
 # Compiled Dirs (http://nodejs.org/api/addons.html)
 build/
-dist/
+client/dist/reviews.js
+client/dist/reviews.js.LICENSE.txt
```

Comment:
A whitelist might be easier, meaning you can ignore client/dist, then allow specific files like index.html.
```
client/dist
!client/dist/index.html
```

Reply:
didnt even know that was a thing, will try it